ci/run_tests.go

```diff
@@ -109,6 +110,11 @@ func recordResults(logger *zap.SugaredLogger, db database.Database, rData *RunDa
 		approvedStatus = pb.Submission_APPROVED
 	}
 
+	// Don't change the build date on a manual rebuild, the date is used as a submission's delivery date
+	if rData.Rebuild {
+		result.BuildInfo.BuildDate = newest.BuildInfo.BuildDate
+	}
```

Comment:
Can you add a test for this?

Reply:
Yup, and I see that `recordResults()` interacts with the database but is generally untested. Should I also add tests for possible outcomes of all those interactions while I'm at it?

Reply:
Yes. More tests are a priority at this point.

Reply:
Added.